src/services/get-songs.ts

```diff
@@ -126,7 +126,7 @@ export default class {
       const {items, nextPageToken} = await this.cache.wrap(
         this.youtube.playlists.items,
         listId,
-        {maxResults: '50', pageToken: nextToken},
+        {maxResults: limit, pageToken: nextToken},
```

Comment:
Playlist limit only applies to Spotify. Here, we're already iterating over all available pages of results for the given YouTube playlist. This should be reverted to the prior behavior of fetching 50 results per page.